forge-gui/res/cardsfolder/upcoming/ashiok_wicked_manipulator.txt

```diff
@@ -0,0 +1,14 @@
+Name:Ashiok, Wicked Manipulator
+ManaCost:3 B B
+Types:Legendary Planeswalker Ashiok
+Loyalty:5
+R:Event$ PayLife | ActiveZones$ Battlefield | ValidPlayer$ You | ReplaceWith$ ExileTop | Amount$ GT0 | Description$ If you would pay life while your library has at least that many cards in it, exile that many cards from the top of your library instead.

```

Comment:
@Hanmac how to limit the condition for the library check?

Reply:
@kevlahnota LEN where where N = Library Size

Reply:
Doesn't work with the amount comparison if I change the condition to Amount$ LEY (SVar Y is declared but crashes since it can't parse the Y variable)

Reply:
fixed